### PR TITLE
Add validation descriptors for RegulationDevice

### PR DESCRIPTION
**Pull request by daniel-thom**
This depends on changes in IS in https://github.com/NREL-SIIP/InfrastructureSystems.jl/pull/98

### CODE REVIEW

File: src/models/generated/FullMachine.jl [hunk 1]
```diff
@@ -99,14 +99,14 @@ function FullMachine(::Nothing)
...
-        L_1d=1.0,
-        L_1q=1.0,
+        L_d=1,
+        L_q=1,
+        L_ad=2,
+        L_aq=2,
+        L_f1d=1,
+        L_ff=2,
+        L_1d=1,
+        L_1q=1,
```
Comment (daniel-thom) [inline src/models/generated/FullMachine.jl:109]: This is an error.  I used `jq` to format the JSON and it converted 2.0 to 2.  I'll fix it

File: src/descriptors/power_system_structs.json [hunk 2]
```diff
@@ -1,5034 +1,5077 @@
...
+          "name": "states",
+          "comment": "",
+          "internal_default": "[:δ, :ω]",
+          "data_type": "Vector{Symbol}"
+        },
+        {
+          "name": "n_states",
+          "comment": "",
+          "internal_default": 2,
+          "data_type": "Int64"
```
Comment (jd-lara) [inline src/descriptors/power_system_structs.json:5047]: max should be 1.0

Comment (jd-lara) [inline src/descriptors/power_system_structs.json:5063]: I am not sure if we can implement (not urgent) that but it should be less than rating in the internal ThermalStandard

**Comment by jd-lara** [comment]
I will correct the values in my branch

**Pull request merged**